Line breaks can't be quoted on Windows

I want to install swiper with npm on a Laravel project.
But I get the following error:

> npm ERR! Line breaks can't be quoted on Windows
>
>npm ERR! A complete log of this run can be found in:
>npm ERR!     C:\Users\hamed\AppData\Local\npm-cache\_logs\2020-11-28T14_07_10_351Z-debug.log

**I tried to install other packages with npm and succeeded**

and this is my error log:
```
0 verbose cli [
0 verbose cli   'C:\\Program Files\\nodejs\\node.exe',
0 verbose cli   'C:\\Users\\hamed\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js',
0 verbose cli   'install',
0 verbose cli   'swiper',
0 verbose cli   '--save-dev'
0 verbose cli ]
1 info using npm@7.0.12
2 info using node@v15.2.1
3 timing config:load:defaults Completed in 2ms
4 timing config:load:file:C:\Users\hamed\AppData\Roaming\npm\node_modules\npm\npmrc Completed in 0ms
5 timing config:load:builtin Completed in 1ms
6 timing config:load:cli Completed in 2ms
7 timing config:load:env Completed in 0ms
8 timing config:load:file:P:\server\www\porsino\.npmrc Completed in 0ms
9 timing config:load:project Completed in 1ms
10 timing config:load:file:C:\Users\hamed\.npmrc Completed in 0ms
11 timing config:load:user Completed in 0ms
12 timing config:load:file:C:\Program Files\nodejs\etc\npmrc Completed in 1ms
13 timing config:load:global Completed in 1ms
14 timing config:load:cafile Completed in 0ms
15 timing config:load:validate Completed in 0ms
16 timing config:load:setUserAgent Completed in 1ms
17 timing config:load:setEnvs Completed in 1ms
18 timing config:load Completed in 9ms
19 verbose npm-session b48091502a798d9b
20 timing npm:load Completed in 20ms
21 timing arborist:ctor Completed in 1ms
22 timing idealTree:init Completed in 675ms
23 timing idealTree:userRequests Completed in 3ms
24 silly idealTree buildDeps
25 silly fetch manifest swiper@*
26 timing arborist:ctor Completed in 0ms
27 http fetch GET 304 https://registry.npmjs.org/swiper 1254ms (from cache)
28 silly placeDep ROOT swiper@6.3.5 OK for:  want: *
29 silly fetch manifest dom7@^3.0.0-alpha.7
30 silly fetch manifest ssr-window@^3.0.0-alpha.4
31 http fetch GET 304 https://registry.npmjs.org/dom7 315ms (from cache)
32 http fetch GET 304 https://registry.npmjs.org/ssr-window 700ms (from cache)
33 timing idealTree:#root Completed in 1966ms
34 silly placeDep ROOT dom7@3.0.0 OK for: swiper@6.3.5 want: ^3.0.0-alpha.7
35 silly placeDep ROOT ssr-window@3.0.0 OK for: swiper@6.3.5 want: ^3.0.0-alpha.4
36 timing idealTree:node_modules/swiper Completed in 6ms
37 timing idealTree:node_modules/dom7 Completed in 0ms
38 timing idealTree:node_modules/ssr-window Completed in 0ms
39 timing idealTree:buildDeps Completed in 1973ms
40 timing idealTree:fixDepFlags Completed in 11ms
41 timing idealTree Completed in 2687ms
42 timing reify:loadTrees Completed in 2688ms
43 timing reify:diffTrees Completed in 30ms
44 silly reify moves {}
45 timing reify:retireShallow Completed in 0ms
46 timing reify:createSparse Completed in 4ms
47 timing reify:loadBundles Completed in 0ms
48 timing reifyNode:node_modules/ssr-window Completed in 70ms
49 timing reifyNode:node_modules/dom7 Completed in 78ms
50 http fetch POST 200 https://registry.npmjs.org/-/npm/v1/security/advisories/bulk 433ms
51 timing auditReport:getReport Completed in 438ms
52 timing auditReport:init Completed in 0ms
53 timing reify:audit Completed in 440ms
54 timing reifyNode:node_modules/swiper Completed in 647ms
55 timing reify:unpack Completed in 647ms
56 timing reify:unretire Completed in 0ms
57 timing build:queue Completed in 1ms
58 info run swiper@6.3.5 postinstall node_modules/swiper echo "[35m[1mLove Swiper? Support Vladimir's work by donating or pledging on patreon:[22m[39m
58 info run  > [32mhttps://patreon.com/vladimirkharlampidi[0m
58 info run "
59 info run swiper@6.3.5 postinstall { code: undefined, signal: undefined }
60 timing reify:rollback:createSparse Completed in 0ms
61 timing reify:rollback:retireShallow Completed in 0ms
62 timing command:install Completed in 3385ms
63 verbose stack Error: Line breaks can't be quoted on Windows
63 verbose stack     at quoteForCmd (C:\Users\hamed\AppData\Roaming\npm\node_modules\npm\node_modules\puka\index.js:41:11)
63 verbose stack     at Formatter.quote (C:\Users\hamed\AppData\Roaming\npm\node_modules\npm\node_modules\puka\index.js:256:12)
63 verbose stack     at quoteForShell (C:\Users\hamed\AppData\Roaming\npm\node_modules\npm\node_modules\puka\index.js:743:34)
63 verbose stack     at C:\Users\hamed\AppData\Roaming\npm\node_modules\npm\node_modules\@npmcli\run-script\lib\make-spawn-args.js:13:90
63 verbose stack     at Array.map (<anonymous>)
63 verbose stack     at escapeCmd (C:\Users\hamed\AppData\Roaming\npm\node_modules\npm\node_modules\@npmcli\run-script\lib\make-spawn-args.js:13:75)
63 verbose stack     at makeSpawnArgs (C:\Users\hamed\AppData\Roaming\npm\node_modules\npm\node_modules\@npmcli\run-script\lib\make-spawn-args.js:37:43)
63 verbose stack     at runScriptPkg (C:\Users\hamed\AppData\Roaming\npm\node_modules\npm\node_modules\@npmcli\run-script\lib\run-script-pkg.js:54:29)
63 verbose stack     at runScript (C:\Users\hamed\AppData\Roaming\npm\node_modules\npm\node_modules\@npmcli\run-script\lib\run-script.js:9:16)
63 verbose stack     at Array.<anonymous> (C:\Users\hamed\AppData\Roaming\npm\node_modules\npm\node_modules\@npmcli\arborist\lib\arborist\rebuild.js:264:17)
64 verbose cwd P:\server\www\porsino
65 verbose Windows_NT 10.0.19042
66 verbose argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\hamed\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "install" "swiper" "--save-dev"
67 verbose node v15.2.1
68 verbose npm  v7.0.12
69 error Line breaks can't be quoted on Windows
70 verbose exit 1
```

I'm having the same issue. I want to install the 6.4.0 version and throw the error `npm ERR! Line breaks can't be quoted on Windows`. Currently, I have the 6.3.5 version installed. The issue mentioned by @vltansky is only pointing to solve this by using yarn, so it is not resolving the particular issue with npm.

--- a/package/package.json
+++ b/package/package.json
@@ -18,7 +18,7 @@
     "swiper-bundle.esm.js"
   ],
   "scripts": {
-    "postinstall": "echo \"\u001b[35m\u001b[1mLove Swiper? Support Vladimir's work by donating or pledging on patreon:\u001b[22m\u001b[39m\n > \u001b[32mhttps://patreon.com/vladimirkharlampidi\u001b[0m\n\""
+    "postinstall": "echo \"\u001b[35m\u001b[1mLove Swiper? Support Vladimir's work by donating or pledging on patreon:\u001b[22m\u001b[39m \" && echo \"> \u001b[32mhttps://patreon.com/vladimirkharlampidi\u001b[0m\""
   },
   "repository": {
     "type": "git",

